src/Razor/test/Microsoft.VisualStudio.LanguageServices.Razor.Test/ProjectSystem/RoslynProjectChangeDetectorTest.cs

```diff
@@ -18,7 +18,7 @@
 
 namespace Microsoft.VisualStudio.Razor.ProjectSystem;
 
-public class WorkspaceProjectStateChangeDetectorTest : VisualStudioWorkspaceTestBase
+public class RoslynProjectChangeDetectorTest : VisualStudioWorkspaceTestBase
```

Comment:
Thanks for updating the test name. We miss that too often, and I really appreciate it!